packages/svelte/src/lib/components/authStore.ts

```diff
@@ -8,16 +8,17 @@ import { interpret } from 'xstate';
 
 import { writable, get } from 'svelte/store';
 
-let _facade = writable<ReturnType<typeof getServiceContextFacade>>(null);
+let _facade = writable(null);
```

Comment:
I'm not sure why but setting this as `<typeof getServiceContextFacade>` is making Svelte remove the `getServiceContextFacade` as if it's a type during the build. This is causing a run time error because Svelte can't find `getServiceContextFacade`. By removing this type, it fixes the error.

Reply:
interesting... 🤯